packages/fxa-auth-server/scripts/test-local.sh

```diff
@@ -16,8 +16,7 @@ npx ts-node ./scripts/gen_keys.js
 npx ts-node ./scripts/gen_vapid_keys.js
 npx ts-node ./scripts/oauth_gen_keys.js
 node ../fxa-auth-db-mysql/bin/db_patcher > /dev/null
-npm run lint:eslint
-grunt copyright
+npm run lint
```

Comment:
Is there a reason why some instances of `npm` get swapped out for `[npx ]yarn <command>` and others are left alone?

Reply:
I'm trying not to go all-in on yarn (for now at least) so I tried to only changed the spots that _needed_ to change. I may not have ended up at the minimum but that was the plan.